torch/_tensor_docs.py

```diff
@@ -614,22 +614,19 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('bernoulli_',
                r"""
-.. function:: bernoulli_(p=0.5, *, generator=None) -> Tensor
+bernoulli_(p=0.5, *, generator=None) -> Tensor
 
-    Fills each location of :attr:`self` with an independent sample from
-    :math:`\text{Bernoulli}(\texttt{p})`. :attr:`self` can have integral
-    ``dtype``.
+Fills each location of :attr:`self` with an independent sample from
+:math:`\text{Bernoulli}(\texttt{p})`. :attr:`self` can have integral
+``dtype``.
 
-.. function:: bernoulli_(p_tensor, *, generator=None) -> Tensor
+:attr:`p` should either be a scalar or tensor containing probabilities to be
+used for drawing the binary random number.
 
-    :attr:`p_tensor` should be a tensor containing probabilities to be used for
-    drawing the binary random number.
-
-    The :math:`\text{i}^{th}` element of :attr:`self` tensor will be set to a
-    value sampled from :math:`\text{Bernoulli}(\texttt{p\_tensor[i]})`.
-
-    :attr:`self` can have integral ``dtype``, but :attr:`p_tensor` must have
-    floating point ``dtype``.
+If it is a tensor, the :math:`\text{i}^{th}` element of :attr:`self` tensor
+will be set to a value sampled from
+:math:`\text{Bernoulli}(\texttt{p\_tensor[i]})`. In this case `p` must have
+floating point ``dtype``.
 
```

Comment:
This docstring was strange: it tried to document the scalar and tensor variants as if they were two different functions. This tripped up autosummary, since it now saw 4 functions (two in the table of contencts, and two in the autosummary generated pages) so it complained.

Reply:
I'm not sure why it documented them separately but I agree with you that we can just merge them together here